src/Routing/RouteBuilder.php

```diff
@@ -692,33 +692,13 @@ public function connect($route, $defaults = [], array $options = []): Route
      * @param string|array $defaults Defaults array from the connect() method.
      * @return array
      */
-    protected static function parseDefaults($defaults): array
+    protected function parseDefaults($defaults): array
```

Comment:
`RouteBuilder` has only non-static methods, `Router` has only static methods.

Method is also invoked with `$this->parseDefaults($target);` since introducing, so it was likely made static erroneously.